sagemaker-python-sdk/dgl_lifesci_bap/pytorch_bap.ipynb

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
<p>Can this output be fetched programmatically? What should the user expect to see and how can it be interpreted and used?</p><p> </p><p>Optionally add a conclusion section summarizing what the user accomplished and recommending any next steps.</p><p> </p><p>If there are AWS resources that will continue incurring cost, please add a Cleanup section to delete them. I don't believe this applies for just a training job as run in this example.</p><p> </p><p>Branding nit: <code>Sagemaker</code>  -&gt; <code>SageMaker</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Added a statement related to the goal of this example and a cell to programmatically download the model.